caluma/caluma_data_source/schema.py

```diff
@@ -33,4 +33,4 @@ def resolve_all_data_sources(self, info, **kwargs):
         return get_data_sources()
 
     def resolve_data_source(self, info, name, **kwargs):
-        return get_data_source_data(info.context.user, name)
+        return get_data_source_data(info.context.user, name, None, None)
```

Comment:
Shouldn't we accept `question` and `context` here as well in the GQL schema? I think we could still make it optional, but depending on the data source, it might not be able to provide data without the new inputs available

Reply:
Good idea, I'll try to add this!

Reply:
Done!